core/Plugin/Menu.php

```diff
@@ -30,16 +30,6 @@
  */
 class Menu
 {
-    protected $module = '';
```

Comment:
Again, the total number of calls doesn't matter since the function returns the plugin name. Called once in 50 plugins is not the same as called 50 times in 1 plugin.

Called once in 50 plugins means the cache would not save anything because it would cache a new value every time.